WalletWasabi/Wallets/WalletFilterProcessor.cs

```diff
@@ -22,7 +22,12 @@ namespace WalletWasabi.Wallets;
 /// <seealso href="https://github.com/zkSNACKs/WalletWasabi/issues/10219">TurboSync specification.</seealso>
 public class WalletFilterProcessor : BackgroundService
 {
-	private const int MaxNumberFiltersInMemory = 1000;
+	private const int MaxNumberFiltersInMemory = 5000;
+
+	// private const int MaxNumberBlocksInMemory = 2500;
```

Comment:
This might be required, see https://github.com/zkSNACKs/WalletWasabi/pull/12148/files#diff-a3e12639ae8c2e59db996ca151cbe714473c840c9bfa4f6da1d8f30508ab1d6eR296-R301